docs/mta/README.md

```diff
@@ -30,6 +31,16 @@ while [[ $retry_count -lt 5 ]]; do
 done
 echo "https://"$(oc -n openshift-mta get route mta -o yaml | yq -r .spec.host)
 ```
+Set the value of `quarkus.rest-client.mta_json.url` to `http://mta-ui.openshift-mta.svc.cluster.local:8080/hub`
+
+The mtaanalysis-props configmap should be similar to this:
+```console
+---
+mta.url = ${MTA_URL:https://<output-of-command-above>}
+quarkus.rest-client.mta_json.url = http://mta-ui.openshift-mta.svc.cluster.local:8080/hub
```

Comment:
Where do you have that?
```
mta.url = ${MTA_URL:http://mta-ui.openshift-mta.svc.cluster.local:8080}
quarkus.rest-client.mta_json.url = ${mta.url}/hub
```
https://github.com/parodos-dev/serverless-workflows/blob/main/mta/application.properties

So, setting `mta.url` should be enough? Is it not?

Reply:
also https://github.com/parodos-dev/serverless-workflows-config/blob/main/charts/mta-v7.x/INSTALL.md#edit-the-mta-analysis-v7-props-configmap